testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -13,6 +13,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
   Scenario: Add SUSE Linux Enterprise Server 12 SP4
     When I follow the left menu "Admin > Setup Wizard > Products"
     And I wait until I do not see "Loading" text
+    And I wait until I do not see "currently running" text
```

Comment:
Actually I think it works in the reverse order right?
First it shows the "...currently running..." message, and after that it starts loading the tree of products and for that it shows a "Loading" message

Reply:
that is a possibility, yes, we only currently see the "currently running" message but the order could be reversed

Reply:
yes, we only see the problem at time of screenshot. But it could be the "Loading" happens after.

I'll invert.

Reply:
Done.